src/style.css

```diff
@@ -1,3 +1,88 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.navi-bar {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.navi-menu {
+  display: flex;
+  list-style: none;
+  margin: 0;
+}
+
+.navi-item {
+  margin: 0;
+  padding-left: 20px;
+}
+
+.navi-item:first-child {
+  padding: 0%;
```

Comment:
This rule will not work as expected because '.navi-item' does not have a default padding that you can reset to 0. You should define padding in '.navi-item' and then override it for the ':first-child' if needed.